app/views/shared/_display_ad_dropdown_content.html.erb

```diff
@@ -0,0 +1,14 @@
+<ul class="list-none m-0">
+    <li>
+        <a class="crayons-link crayons-link--block" href='/billboard'>
+            <%= crayons_icon_tag("info", aria_hidden: true, class: "c-btn__icon") %>
+            <%= I18n.t("display_ads.what_is_a_billboard") %>
+        </a>
+    </li>
+    <li>
+        <a class="crayons-link crayons-link--block" href='/settings/customization'>
```

Comment:
One possible enhancement here (or later) might be if we had `#anchors` on the settings page, we could link directly to the sponsorship settings (since, otherwise, the user might not realize where on that page the settings could be found, and text search isn't helpful).

Reply:
Done. I have added `id=sponsors" to the destination content and updated this URL accordingly.